tests/namespace/test_arp.py

```diff
@@ -0,0 +1,167 @@
+import os
+import sys
+import importlib
+
+modules_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.insert(0, os.path.join(modules_path, 'src'))
+
+from unittest import TestCase
+from unittest.mock import patch, mock_open
+
+# noinspection PyUnresolvedReferences
+import tests.mock_tables.dbconnector
+import tests.mock_tables.python_arptable
+from ax_interface.mib import MIBTable
+from ax_interface.pdu import PDUHeader
+from ax_interface.pdu_implementations import GetPDU, GetNextPDU
+from ax_interface import ValueType
+from ax_interface.encodings import ObjectIdentifier
+from ax_interface.constants import PduTypes
+from sonic_ax_impl.main import SonicMIB
+from sonic_ax_impl.mibs.ietf import rfc1213
+class TestSonicMIB(TestCase):
+    @classmethod
+    def setUpClass(cls):
+        tests.mock_tables.python_arptable.arp_filename = '/host_arp.txt'
+        tests.mock_tables.dbconnector.load_namespace_config()
+        importlib.reload(rfc1213)
+        cls.lut = MIBTable(rfc1213.IpMib)
+        for updater in cls.lut.updater_instances:
+            #updater.update_data()
+            updater.reinit_data()
+            updater.update_data()
+
+    # TODO: this test fails, possible reason maybe mock arptable
+    # def test_update(self):
+        # for updater in self.lut.updater_instances:
+            # updater.update_data()
+            # updater.reinit_data()
+            # updater.update_data()
```

Comment:
Is it working?
Or remove it?

Reply:
removed it.

Reply:
Not yet?

---
In reply to: [451194502](https://github.com/Azure/sonic-snmpagent/pull/129#discussion_r451194502) [](ancestors = 451194502)

Reply:
It is not there in the latest file.
https://github.com/Azure/sonic-snmpagent/pull/129/commits/d728e97611d1a561ad5ff2277f61dcd46994676a